File: python/G.py
def main():
    expresion = input()
    
    #Opcion 1
    resultado = ''.join(dict.fromkeys(expresion))
    
    # Opcion 2
    # resultado = str()
    # for letra in expresion:
    #     if(letra not in resultado):
    #         resultado += letra
        
    print(resultado)

if(__name__ == '__main__'):
    main()

File: java/G.java
import java.util.Scanner;

public class G{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        String expresion = sc.nextLine();
        String resultado = "";
        String letraStr;
        for (char letra : expresion.toCharArray()) {
            letraStr = String.valueOf(letra);
            if(!resultado.contains(letraStr)){
                resultado = resultado.concat(letraStr);
            }
        }

        System.out.println(resultado);
        sc.close();
    }
}